docs/src/api/class-tracing.md

```diff
@@ -11,6 +15,8 @@ const context = await browser.newContext();
 await context.tracing.start({ screenshots: true, snapshots: true });
 const page = await context.newPage();
 await page.goto('https://playwright.dev');
+// Not captured
```

Comment:
Again, "not captured" is not really helpful here. Either say "see note above", or just drop it from the snippet. I think the note is enough.